actions/remove_empty_dirs.py

```diff
@@ -323,8 +322,8 @@ def empty_action(storage_path, rse, out, lfn_converter, stats, stats_key, dry_ru
 
 storage_path, rse = args
 
-config = EmptyActionConfiguration(rse, opts["-c"])
-scanner_config = ScannerConfiguration(rse, opts["-c"])
+config  = ActionConfiguration(rse, opts["-c"], "dark")
```

Comment:
Why is "dark" used as an `action` here?

Reply:
Ah, found my answer in your comment above. 
```
    dark_action:
      confirmation_window: 35
      max_age_last_run: 6
      max_fraction: 0.01
      min_age_first_run: 18
      min_runs: 3
```

yes, we can add a new section for empty configuration.